### PR TITLE
Manager add stress read cmd into ipv6 config file

### PR DESCRIPTION
**Pull request by mikliapko**
The PR addresses the following issues:
- Configuration yaml for manager ipv6 test is missing stress_read_cmd parameter. Without it, read verification check is skipped in test.

### Testing

- [x] [backup-test](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/manager-master/job/sct-feature-test-backup/4/)
- [x] [sanity-ipv6](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-sanity-ipv6-test/4/)
- [x] [ubuntu22-sanity](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-sanity-test/26/)
- [x] [vnodes-tablets-sanity](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-vnodes-tablets-cluster-test/15/)

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Review by rayakurl — APPROVED** [review]
LGTM

**Comment by mikliapko** [comment]
@scylladb/qa-maintainers @karol-kokoszka @Michal-Leszczynski 
Guys, could you please take a look? Wanna merge it before going to PTO to unblock this test.

**Review by Michal-Leszczynski — APPROVED** [review]

File: mgmt_cli_test.py
Comment (vponomaryov) [inline mgmt_cli_test.py:185]:
```
Fixed list mutation issue in verify_backup_success method. 
The previous implementation changed the initial lists ks_names and tables_names...
```
I don't see mutation of the original object here.
I see a change of behavior for empty input list.
When the input value is `None` then we create new `list` object and don't mutate the immutable `None`.

```
. Since the same list was used in the caller method, it resulted in unintended behavior.
```

The diff is in the behavior in this method, why the commit/PR refers to a caller method?
If the point of this commit/PR is in the change of the empty list behavior then exactly this should be stated.
For now it is confusing.

**Review by karol-kokoszka — APPROVED** [review]

**Review by vponomaryov — APPROVED** [review]
LGTM

**Review by ilya-rarov — APPROVED** [review]
LGTM

**Pull request merged**
